### ISSUE TITLE
Plots not showing when called from within eval

### ISSUE THREAD

**Issue by prbuen** [blink1073/oct2py#158]
Hi,

I have just installed oct2py and am trying to make plots from inline octave source. When I use`Oct2Py.plot`, a figure with the plot pops up, but not when I call octave's plot function from within `Oct2Py.eval`:
```python
from oct2py import octave
# this works fine:
octave.plot([1,2,3],'-o', 'linewidth', 2) 

# this doesn't work:
octave.eval("""\
plot([1,2,3], '-o', 'linewidth', 2)
""")
```
There is no error message when running `eval`. The code executes silently -- no figure window is opened. 

I am on MacOS 10.14, python 3.7.4, octave 5.1.0, oct2py 5.0.4, using gnuplot 5.2.7 (term x11). I tried to play with a few settings, but nothing makes a difference. Any idea what could be going on, or how I could troubleshoot this?

Thanks!

**Comment by blink1073** [comment]
Hi @prbuen, I don't have the bandwidth to make changes to this repo at the moment, but it looks like some of the plot handling logic in [feval](https://github.com/blink1073/oct2py/blob/564fad5dfdb2e7985fc46c0407e32b78a9f5046e/oct2py/core.py#L346) would need to be brought into [eval](https://github.com/blink1073/oct2py/blob/564fad5dfdb2e7985fc46c0407e32b78a9f5046e/oct2py/core.py#L381)

**Comment by prbuen** [comment]
Indeed, writing the octave source code to a `.m` script and calling it with `feval` works as expected. It would be great if that plot handling code could be merged into `eval`, thanks!
